auto/cc/test

```diff
@@ -45,19 +45,10 @@ then
     NXT_CC_VERSION=`/bin/sh -c "($CC -v)" 2>&1 | grep "Apple LLVM version" 2>&1`
     $echo " + $NXT_CC_VERSION"
 
-else
-if `/bin/sh -c "($CC -V)" 2>&1 | grep "Sun C" >> $NXT_AUTOCONF_ERR 2>&1`
-then
-    NXT_CC_NAME=SunC
-    $echo " + using Sun C compiler"
-    NXT_CC_VERSION=`/bin/sh -c "($CC -V)" 2>&1 | grep "Sun C" 2>&1`
-    $echo " + $NXT_CC_VERSION"
-
 else
     NXT_CC_NAME=cc
     NXT_CC_VERSION=cc
 
-fi # SunC
```

Comment:
I would also

```diff
diff --git a/auto/cc/test b/auto/cc/test
index 499a2fb1..785da428 100644
--- a/auto/cc/test
+++ b/auto/cc/test
@@ -74,7 +74,6 @@ case $NXT_CC_NAME in
         # Do not export symbols except explicitly marked with NXT_EXPORT.
         NXT_CFLAGS="$NXT_CFLAGS -fvisibility=hidden"
 
-        # c99/gnu99 conflict with Solaris XOPEN.
         #NXT_CFLAGS="$NXT_CFLAGS -std=gnu99"
 
         NXT_CFLAGS="$NXT_CFLAGS -O"
```

If so, then

```
Co-developed-by: Alejandro Colomar <alx@kernel.org>
Signed-off-by: Alejandro Colomar <alx@kernel.org>
```

Reply:
Ahh, no, that's a conflict with solaris libc, not cc.  Ignore this.

Reply:
Ah, that will be the next card I play! Which is your gnu11 [patch](https://github.com/nginx/unit/pull/1012/commits/0c9c0b90f92b361a7121888ee9d2854bb3f5dc68).

Reply:
I will have been able to get more things fixed as an external contributor than when being paid to do so.  lol